src/introspection/memory/ffi.rs

```diff
@@ -123,11 +123,12 @@ pub struct MemPoolIntrospectionTopic {
 }
 
 impl MemPoolIntrospectionTopic {
-    pub fn new() -> Topic<Self> {
-        TopicBuilder::<Self>::new("Introspection", "RouDi_ID", "MemPool")
+    pub fn new() -> InactiveSubscriber<Self> {
+        SubscriberBuilder::<Self>::new("Introspection", "RouDi_ID", "MemPool")
             .queue_capacity(1)
             .history_request(1)
-            .build()
+            .create_without_subscribe()
+            .expect("Create subscriber")
```

Comment:
Isn't this the error message when something goes wrong? It seems like creating a subscriber is an error - what about "failed to create subscriber"?

Reply:
I think I will also return a `Result` in a follow up PR but this usually tells what was expected. In this case it was expected to create a subscriber and not to fail to create a subscriber.

The error message will look like this
```
thread 'main' panicked at 'Create subscriber', src/introspection/memory/ffi.rs:131:14
```